src/com/oltpbenchmark/DBWorkload.java

```diff
@@ -1039,11 +1039,16 @@ private static Results runWorkload(List<BenchmarkModule> benchList, boolean verb
       xmlConfig.getBoolean("displayEnhancedLatencyMetrics");
     PrintLatencies(workers, displayEnhancedLatencyMetrics);
 
-    int total_retries = 0;
+    int[] totalRetries = {0, 0, 0, 0, 0};
+    int[] totalFailures = {0, 0, 0, 0, 0};
     for (Worker<?> w : workers) {
-      total_retries += w.getTotalRetries();
+      for (int i = 0; i < 5; ++i) {
```

Comment:
can we avoid magic numbers like 5?

Reply:
i think we can use `getNumTxnTypes` in `WorkloadConfiguration`

Reply:
Done.